python-sdk/src/astro/files/types/excel.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import io
```

Comment:
Looking good. Just one thing - can we change the dir structure? if it's not too much trouble?
it can be something like:
 python-sdk/src/astro/files/types/excel
python-sdk/src/astro/files/types/excel/base.py
python-sdk/src/astro/files/types/excel/xls.py
python-sdk/src/astro/files/types/excel/xlsx.py

WDYT?

Reply:
I thought about it but some test refactoring requires

Reply:
maybe we can do it as separate PR if needed